pagopa/pn-frontend
# feat: add privacy policy for pn-personafisica-login

## Short description
Adds privacy policy page in the login app

## List of changes proposed in this pull request
- Adds privacy policy page in pn-personafisica-login

## How to test
Go to the route /informativa-privacy and check that privacy policy and cookie policy are present
